frontend/public/components/_list.scss

```diff
@@ -13,11 +13,6 @@
   }
 }
 
```

Comment:
+1 for removing, `middler` is a very vague classname. :)

Reply:
`middler` looked to be used randomly and set `display:flex; align-items: center`
`.co-resource-list-item` now controls that.